pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -25,15 +25,15 @@ type mlnxNic struct {
 }
 
 const (
-	EthLinkType           = "ETH"
-	InfinibandLinkType    = "IB"
-	PreconfiguredLinkType = "Preconfigured"
```

Comment:
Is this replaced by `StaticLinkType`? or does it only applied to certain kind of devices?

Reply:
It is just replacing of the variable, not related to devices